src/config/__snapshots__/config.unit.js.snap

```diff
@@ -160,7 +170,7 @@ exports[`Configuration Verify app-common-js function 1`] = `
   },
   "validateResponseStrict": true,
   "vmaas": {
-    "host": "https://console.redhat.com",
+    "host": "https://webapp-vmaas-prod.apps.crcp01ue1.o9m8.p1.openshiftapps.com",
```

Comment:
I don't recall changing this, just checking, if you did that's fine but if not, maybe we can revert it.